homeassistant/components/switch/rainmachine.py

```diff
@@ -42,10 +43,12 @@
             vol.Email(),  # pylint: disable=no-value-for-parameter
             vol.Required(CONF_PASSWORD):
             cv.string,
+            vol.Optional(CONF_PORT):
+            cv.port,
+            vol.Optional(CONF_SSL):
+            cv.boolean,
             vol.Optional(CONF_ZONE_RUN_TIME, default=DEFAULT_ZONE_RUN_SECONDS):
-            cv.positive_int,
-            vol.Optional(CONF_HIDE_DISABLED_ENTITIES, default=True):
```

Comment:
Not really:

1. The functionality of this parameter (a) was acting poorly with the device and (b) didn't provide any tangible benefit.
2. This parameter was disabled and undocumented to begin with.
3. Based on the changes made at the bottom of `async_setup_platform`, even if a user leaves this option in the config, nothing will break.